app/src/main/java/com/ncnf/map/ui/MapFragment.java

```diff
@@ -85,12 +79,12 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         materialSearchBar = getView().findViewById(R.id.searchBarMap);
 
         // Initialize Google Map with the callback onMapReady
-        mapView = (MapView) getView().findViewById(R.id.map);
+        mapView = getView().findViewById(R.id.map);
         mapView.onCreate(savedInstanceState);
         mapView.getMapAsync(googleMap -> {
             mMap = googleMap;
 
-            mapHandler = new MapHandler(getActivity(), mMap, eventDB, venueProvider, getChildFragmentManager());
+            mapHandler = new MapHandler((AppCompatActivity) getActivity(), mMap, venueProvider, getChildFragmentManager());
```

Comment:
you can also use requireActivity() for the same reasons stated above

Reply:
Same, will do!